stdlib/public/Reflection/TypeLowering.cpp

```diff
@@ -258,13 +258,16 @@ BuiltinTypeInfo::readExtraInhabitantIndex(remote::MemoryReader &reader,
 bool RecordTypeInfo::readExtraInhabitantIndex(remote::MemoryReader &reader,
                                               remote::RemoteAddress address,
                                               int *extraInhabitantIndex) const {
+  *extraInhabitantIndex = -1;
+
   switch (SubKind) {
   case RecordKind::Invalid:
   case RecordKind::ClosureContext:
     return false;
 
-  case RecordKind::OpaqueExistential: {
-    if (Fields.size() != 1) {
+  case RecordKind::OpaqueExistential:
+  case RecordKind::ExistentialMetatype: {
+    if (Fields.size() < 1) {
```

Comment:
I'm not sure how "Fields" works here.  Does it count the associated type/protocol conformance information as well?  Otherwise, I'm not sure how it could ever be greater than 1.

Reply:
I don't think it is, right now, but if someone added something, the `!= 1` test would fail, which would probably be wrong.